src/views/Timeline.vue

```diff
@@ -11,36 +11,39 @@ import { useProfiles } from '@/composables/useProfiles';
 import { useFollowSpace } from '@/composables/useFollowSpace';
 import { useWeb3 } from '@/composables/useWeb3';
 import verified from '@/../snapshot-spaces/spaces/verified.json';
+import zipObject from 'lodash/zipObject';
+import { useStore } from '@/composables/useStore';
 
-const verifiedSpaces = Object.entries(verified)
-  .filter(space => space[1] === 1)
-  .map(space => space[0]);
+const { store } = useStore();
 
-import zipObject from 'lodash/zipObject';
-const filterBy = ref('all');
 const loading = ref(false);
-const proposals = ref([]);
 
 const route = useRoute();
 const { followingSpaces, loadingFollows } = useFollowSpace();
 const { web3 } = useWeb3();
 
-const following = computed(() => {
-  return route.name === 'timeline' ? followingSpaces.value : verifiedSpaces;
+const spaces = computed(() => {
+  const verifiedSpaces = Object.entries(verified)
+    .filter(space => space[1] === 1)
+    .map(space => space[0]);
+  if (route.name === 'timeline') return followingSpaces.value;
+  if (route.name === 'explore') return verifiedSpaces;
+  else return [];
 });
 
-const isTimeline = computed(() => route.name === 'timeline');
-
-watch(following, () => {
-  proposals.value = [];
-  load();
+watch(spaces, () => {
+  if (route.name === 'timeline' || route.name === 'explore') {
```

Comment:
This is always true. Those are the two routes this view is used for. And if there will be other routes, I guess the behavior should still be the same. So, not sure how much this if here is needed.

Reply:
This is only to prevent reloading when switching to another route not related to timeline.

Reply:
f.e when you click on a proposal in the timeline it will somehow still trigger this watcher and when you click back in the browser you will see a different list. Try removing it, maybe there is a better solution?